components/resources/resource-card.tsx

```
@@ -0,0 +1,131 @@
+                <Button
+                  variant="ghost"
+                  size="icon"
+                  className={`h-7 w-7 ${resource?.upvoteCount ? "text-primary dark:text-primary" : "text-slate-500 dark:text-slate-400"}`}
+                  onClick={handleUpvote}
+                >
+                  <ArrowUp size={14} />
+                </Button>
+                <Button
+                  variant="ghost"
+                  size="icon"
+                  className={`h-7 w-7 ${true ? "text-primary dark:text-primary" : "text-slate-500 dark:text-slate-400"}`}
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)